assets/sass/_pricing.scss

```diff
@@ -1,31 +1,137 @@
-.pricing-item-heading {
-    @apply bg-gray-200 text-purple-700 text-sm font-display font-bold border-b border-gray-300 px-2 py-4;
+.pricing-plans {
+    &-item-container {
+        @apply px-4 text-center;
 
-    @screen md {
-        @apply text-base p-4;
+        @screen lg {
+            @apply w-1/3;
+        }
+    }
+
+    &-item-header {
+        @apply text-purple-500 font-bold text-lg mb-5;
+    }
+
+    &-item-description {
+        @apply text-sm text-gray-700 mt-0 mb-2 px-3;
+    }
+
+    &-item-price {
+        @apply text-4xl text-black font-bold mt-0;
+    }
+
+    &-item-info-title {
+        @apply text-sm font-bold;
     }
 }
 
-.pricing-item {
-    @apply flex items-center border-b border-gray-100 text-xs text-center;
+.pricing-compare-plans-table {
+    @apply mt-8;
+
+    .compare-plans-table-header {
+        &-container {
+            @apply flex px-8 py-5;
+            background-color: rgba(42, 19, 55, 0.05);
```

Comment:
Let's make this either `bg-gray-100` or `-200`, to keep it within the palette. (Alternatively, we could extend that palette to include another hue, but I'd rather not do one-off styles like this unless they're not really representable with Tailwind.)